src/zero/service/pubsub.clj

```diff
@@ -7,6 +7,15 @@
             [zero.once         :as once])
   (:import [java.util Base64]))
 
+(comment
```

Comment:
I'd make this block of comments a multi-line `;;` comment simply because most IDEs can grey out `;;` to make them less distractive, but this is totally personal preference, so it's up to you!

Reply:
NVM, this is on an old commit.